frontend/lib/src/components/elements/GraphVizChart/GraphVizChart.tsx

```diff
@@ -65,7 +65,7 @@ export function GraphVizChart({
 
   return (
     <StyledGraphVizChart
-      className="graphviz stGraphVizChart"
```

Comment:
sanity-check: was `graphviz` maybe used by the graphviz library to apply some styling? just asking because the `className` sounds like an official one and I am wondering what the reason was to include it besides our own `className: stGraphVizChart`

Reply:
I was also wondering about that. But our graphviz examples render fine.

Reply:
alrighty, if our stuff renders fine and no test catches it, we should clean it up!